module/data/activity/base-activity.mjs

```diff
@@ -206,16 +210,39 @@ export default class BaseActivityData extends foundry.abstract.DataModel {
    * @returns {FormSelectOption[]}
    */
   static validItemUsesTargets() {
+    const makeLabel = (name, item) => {
+      let label;
+      const uses = item.system.uses;
+      if ( uses.max && (uses.recovery?.length === 1) && (uses.recovery[0].type === "recoverAll") ) {
+        const per = CONFIG.DND5E.limitedUsePeriods[uses.recovery[0].period]?.abbreviation;
+        label = game.i18n.format("DND5E.AbilityUseConsumableLabel", { max: uses.max, per });
+      }
+      else if ( uses.value ) label = game.i18n.format("DND5E.AbilityUseChargesLabel", { value: uses.value });
```

Comment:
We should probably still use the charges label even if there are no charges currently left.